pom.xml

```diff
@@ -0,0 +1,200 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.7.7</version>
+        <relativePath/>
+    </parent>
+
+
+    <groupId>io.github.dftrakesh</groupId>
+    <artifactId>woocommerce</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>woocommerce</name>
+    <description>woocommerce API using JDK 11</description>
+    <url>https://github.com/dftrakesh/woocommerce</url>
+
+    <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>Rakesh Dobariya</name>
+            <email>rakesh@dakshyafreondtech.com</email>
+            <organization>Dakshya Freondtech</organization>
+            <organizationUrl>https://dakshyafreondtech.com/</organizationUrl>
+        </developer>
+    </developers>
+
+    <scm>
+        <url>https://github.com/dftrakesh/woocommerce</url>
+    </scm>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <version.plugin.maven-release-plugin>2.5.2</version.plugin.maven-release-plugin>
+        <version.plugin.nexus-staging-maven-plugin>1.6.5</version.plugin.nexus-staging-maven-plugin>
+        <version.plugin.maven-gpg-plugin>1.6</version.plugin.maven-gpg-plugin>
+        <source.encoding>UTF-8</source.encoding>
+    </properties>
+
+    <dependencies>
```

Comment:
Why do you need spring boot starter?

Reply:
I Started Work on next Ticket before Submitting PR. In That I need @Controller So for that i added that

Reply:
The next ticket is for working on another module. the controller will never come in sdk.

Reply:
Ok i will remove it